velox/docs/functions/spark/string.rst

```diff
@@ -59,6 +59,25 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
         SELECT ltrim('ps', 'spark'); -- "ark"
 
+.. spark:function:: overlay(input, replace, pos, len) -> same as input
+
+    Replace a substring of ``input`` starting at ``pos`` character with ``replace`` and
+    going for rest ``len`` characters of ``input``.
+    ``input`` has same type with ``replace``, can be varchar or varbinary, when ``input`` is varchar,
```

Comment:
Split this into multiple sentences for readability.

```
Types of input and replace must be the same. Supported types are: VARCHAR and VARBINARY.

When input types are VARCHAR, len and pos are specified in characters, otherwise, bytes.
```